link/kprobe.go

```diff
@@ -413,14 +415,15 @@ func createTraceFSProbeEvent(typ probeType, args probeArgs) error {
 		// p:ebpf_5678/main_mySymbol /bin/mybin:0x12345(0x123)
 		//
 		// See Documentation/trace/uprobetracer.txt for more details.
-		pe = fmt.Sprintf("%s:%s/%s %s", probePrefix(args.ret), args.group, args.symbol, uprobeToken(args))
+		token = uprobeToken(args)
+		pe = fmt.Sprintf("%s:%s/%s %s", probePrefix(args.ret), args.group, args.symbol, token)
 	}
 	_, err = f.WriteString(pe)
 	// Since commit 97c753e62e6c, ENOENT is correctly returned instead of EINVAL
 	// when trying to create a kretprobe for a missing symbol. Make sure ENOENT
 	// is returned to the caller.
-	if errors.Is(err, os.ErrNotExist) || errors.Is(err, unix.EINVAL) {
-		return fmt.Errorf("symbol %s not found: %w", args.symbol, os.ErrNotExist)
+	if errors.Is(err, os.ErrNotExist) || errors.Is(err, unix.EINVAL) || errors.Is(err, syscall.ERANGE) {
```

Comment:
Nit: can you split the syscall.ERANGE check into a separate if and add a comment that explains it?

Also, maybe it's better to return `sys.Error(os.ErrNotExist, syscall.ERANGE)` here? Then checks for both ErrNotExist and ERANGE will succeed, but we keep the additional context from ERANGE.

Reply:
I've tried using `sys.Error()` but there's way too much unwrapping (4x IIRC) to do in order to test errno. Am I doing it the wrong way? 

```
        if errors.Is(err, syscall.EILSEQ) {
-               return nil, fmt.Errorf(snip: %w, os.ErrNotExist)
+               err = sys.Error(os.ErrNotExist, syscall.EILSEQ)
+               return nil, fmt.Errorf(snip: %w, err)
        }
```